core/src/test/java/org/apache/calcite/test/SqlToRelConverterTest.java

```diff
@@ -3136,6 +3136,19 @@ void checkCorrelatedMapSubQuery(boolean expand) {
     sql(sql).ok();
   }
 
+  @Disabled("CALCITE-985")
```

Comment:
So we can't run this test yet?

Reply:
Seems like it. Essentially, there's existing validation issues in calcite for MERGE INTO:
https://issues.apache.org/jira/browse/CALCITE-985

This issue is fairly recent (2019), and I could find no other tests that test merge into, so I'll likely have to deal with this.

Reply:
Okay makes sense. Thanks.